package_schemas/general_data/2_restricted_schema.json

```diff
@@ -51,6 +51,14 @@
             "private": false,
             "visibility_classes": ["hidden"]
         },
+        {
+            "field_name": "org_to_allow_transfer_to",
+            "label": "Transfer to",
+            "preset": "org_to_allow_transfer_to",
+            "private": false,
```

Comment:
This is a field I take it that you have copied across from the organisation field?  It is specific to the organisation form snippet.  Are you sure you need it for your feature?